tests/laser_blowout_wake_explicit.1Rank.sh

```diff
@@ -29,7 +29,9 @@ mpiexec -n 1 $HIPACE_EXECUTABLE $HIPACE_EXAMPLE_DIR/inputs_normalized \
         beams.names = no_beam \
         geometry.prob_lo     = -20.   -20.   -8  \
         geometry.prob_hi     =  20.    20.    6  \
+        laser.use_laser = 1 \
```

Comment:
Let's remove this once the default is set automatically

Reply:
I think it's a bit cleaner this way.